persistence/src/main/java/com/epam/esm/dao/api/GiftCertificateDao.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.esm.dao.api;
```

Comment:
As I see we have dao classes with the same structure of methods -> could we create general interface for all dao classes?

Reply:
No, we can't, because not all dao methods have the same methods (for example UserDao class has only two get methods)